### PR TITLE
[DO-1750] Create PEP18 scoring rule

### PR DESCRIPTION
**Pull request by Andrii17s**
Rule 18
Declared ownership and/or right of use of a business class car, or car with a price exceeding 800 000 UAH or brand vehicle, which is considered to be a luxury car

### CODE REVIEW

File: business_register/pep_scoring/rules.py [hunk 1]
```diff
@@ -104,3 +104,67 @@ def calculate_weight(self):
...
+                "type": 'owner',
+                "reason": 'luxury',
+                "declaration_id": have_car[0][0],
+            }
+            return weight, data
+        have_car = Vehicle.objects.filter(
+            declaration__pep_id=self.pep.id,
+            valuation__gt=max_price,
+        ).values_list('declaration_id', 'valuation')[::1]
+        print(have_car)
```
Comment (roman-tiukh) [inline business_register/pep_scoring/rules.py:135]: remove prints

File: business_register/pep_scoring/rules.py [hunk 2]
```diff
@@ -104,3 +104,67 @@ def calculate_weight(self):
...
+            weight = 0.4
+            data = {
+                "type": 'owner',
+                "reason": 'luxury',
+                "declaration_id": have_car[0][0],
+            }
+            return weight, data
+        have_car = Vehicle.objects.filter(
+            declaration__pep_id=self.pep.id,
+            valuation__gt=max_price,
```
Comment (roman-tiukh) [inline business_register/pep_scoring/rules.py:133]:
```suggestion
            valuation__gte=max_price,
```

File: business_register/pep_scoring/rules.py [hunk 3]
```diff
@@ -104,3 +104,67 @@ def calculate_weight(self):
...
+    weight - 0.4
+    Declared ownership and/or right of use of a business class car, or car with a price exceeding
+    800000 UAH or brand vehicle, which is considered to be a luxury car
+    """
+
+    def calculate_weight(self):
+        max_price = 800000  # max price of non-luxury vehicle
+        have_car = Vehicle.objects.filter(
+            declaration__pep_id=self.pep.id,
+            is_luxury=True
```
Comment (roman-tiukh) [inline business_register/pep_scoring/rules.py:121]: how do we calculate luxury?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/pep_scoring/rules.py (hunk 3 above)
Comment (Andrii17s) [reply business_register/pep_scoring/rules.py:121]: It based on pep.org.ua data, until we haven't our method yet.

**Comment by criticus** [comment]
@EmmaSamoilova should this one be closed?

**Comment by EmmaSamoilova** [comment]
> @EmmaSamoilova should this one be closed?

maybe yes, I think I've finished this rule.

**Pull request closed without merging**